src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, mgr):
         # volume specification
         self.volspec = VolSpec(mgr.rados.conf_get('client_snapdir'))
         self.cloner = Cloner(self, self.mgr.max_concurrent_clones, self.mgr.snapshot_clone_delay)
+        self.clone_reporter = CloneProgressBar(self)
```

Comment:
In this (separate) commit, I see this has been made a (singleton) object - which is correct.